setup.cfg

```diff
@@ -4,28 +4,17 @@ universal = 1
 [flake8]
 max-line-length = 119
 ignore =
-    # This rule goes against the PEP 8 recommended style and it's incompatible
-    # with W504
     W503
 
-    # Exclude automatically generated files
-    # E501: Line too long
-    dateparser/data/date_translation_data/* E501
-
-    # Exclude files that are meant to provide top-level imports
-    # F401: Module imported but unused
-    dateparser/data/__init__.py F401
-    dateparser/languages/__init__.py F401
-
-    # Issues pending a review:
-    dateparser/freshness_date_parser.py E722
-    dateparser/parser.py E722
-    dateparser/docs/conf.py E402
-
-    # Additional ignored codes
     E203
     E501
     E722
     F401
     E701
     E704
+
+exclude =
+    dateparser/data/date_translation_data/*
+    dateparser/data/__init__.py
+    dateparser/languages/__init__.py
+    docs/conf.py
```

Comment:
In the pipeline, I got some errors relative to flake, after changing no errors anymore